modules/openshift-cluster-maximums-environment.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc
+
+[id="cluster-maximums-environment_{context}"]
+= {product-title} environment and configuration on which the cluster maximums are tested
+
+AWS cloud platform:
+
+[options="header",cols="8*"]
+|===
+| Node |Flavor |vCPU |RAM(GiB) |Disk type|Disk size(GiB)/IOS |Count |Region
+
+| Master/Etcd footnoteref:[masteretcdnode, io1 disks with 3000 IOPS are used for master/etcd nodes as etcd is i/o intensive and latency sensitive.]
+| r5.4xlarge
+| 16
+| 128
+| io1 
+| 220 / 3000
+| 3
+| us-west-2
+
+| Infra footnoteref:[infranodes,Infra nodes are used to host openshift-monitoring, ingress and router components to make sure they have enough resources to run at large scale. Procedure on creating Infra machinesets can be found at https://docs.openshift.com/container-platform/4.3/machine_management/creating-infrastructure-machinesets.html.]
```

Comment:
@ahardin-rh how can we reuse the footnoteref in case the text is the same instead if repeating it? For example, infranodes footnoteref is the same for both AWS and Azure cloud platforms.

Reply:
@chaitanyaenr Looks like you can just re-use the ID per https://asciidoctor.org/docs/user-manual/#user-footnotes; we may have to just play with it to see how it impacts the build.

Reply:
Infra nodes are used to host openshift-monitoring, Ingress, and router components to ensure they have enough resources to run at a large scale.

Unfortunately, we have to remove the link to the procedure, as we are currently not allowed to include internal xrefs within modules. Perhaps we can move this to a new "Related Information" section of the "assembly" file (as seen in [this template](Related Information)), which in this case is the planning-your-environment-according-to-object-maximums.adoc.